src/ProjectTemplates/Quantum.App1/Quantum.App1.csproj.v.template

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.Quantum.Sdk/#NUGET_VERSION#" InitialTargets="RecompileOnChange">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <QsharpCompile Include="Program.qs" />
```

Comment:
Can we keep the default `**/*.qs` item to avoid needing to explicitly list Q# sources? Eventually it would be good to just have "Compile" here, but I suspect that would require moving to a qsproj...

Reply:
I added the corresponding item group (with **/*.qs) to the project templates. I can't seem to have it in the sdk, since then it is not possible to remove them (at least not in VS). I'm open to suggestions, but I think having it in the template is ok.